src/components/DpCheckboxGroup/DpCheckboxGroup.vue

```diff
@@ -35,6 +36,11 @@ export default {
   },
 
   props: {
+    dataCy: {
+      type: String,
+      required: false
```

Comment:
Why didn't you define a default here (maybe empty string)?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/773/commits/37ee71499d3b44f1b034822ab5b4d658e6330227